proof/refine/RISCV64/ArchInvsLemmas_H.thy

```diff
@@ -519,7 +519,7 @@ locale typ_at_props' =
   assumes typ': "f \<lbrace>\<lambda>s. P (typ_at' T p' s)\<rbrace>"
 begin
 
-interpretation Arch . (*FIXME RT arch-split *)
+interpretation Arch . (* FIXME arch-split RT *)
```

Comment:
same question as the other RT-tagged `interpretation Arch .`, plus one more in this file

Reply:
oh wait, this one's a locale; so the question is only why is the tag RT, RT-only locale?